docs/examples/Make Test Dataset.ipynb

```diff
@@ -98,7 +98,7 @@
     "### JSON Data Format\n",
     "\n",
     "The data format is well explained in these resources\n",
-    "[Serialization documentation](../user_manual/serialization.md) and some examples of Serialization are given in [Serialization notebook](./Serialization%20Example.ipynb)"
+    "[Serialization documentation](https://power-grid-model.readthedocs.io/en/stable/user_manual/serialization.html#serialization) and some examples of Serialization are given in [Serialization notebook](./Serialization%20Example.ipynb)"
```

Comment:
please revert this. you will need to automate adding the `https://power-grid-model.readthedocs.io/en/stable` prefix and making `md` => `html` in the workshop repo